ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -1190,6 +1204,79 @@ objects:
             rm -f /tmp/artifacts/installer/openshift/99_openshift-cluster-api_worker-machineset-*.yaml
             sed -i "s;mastersSchedulable: true;mastersSchedulable: false;g" /tmp/artifacts/installer/manifests/cluster-scheduler-02-config.yml
 
+        elif [[ "${CLUSTER_TYPE}" == "openstack" ]]; then
+            # We have to truncate cluster name to 14 chars, because there is a limitation in the install-config
+            # Now it looks like "ci-op-rl6z646h-65230".
+            # We will remove "ci-op-" prefix from there to keep just last 14 characters.
+            export CLUSTER_NAME=${CLUSTER_NAME:${#CLUSTER_NAME}<14?0:-14}
+
+            # create api and ingress FIPS
+            LB_FIP_AND_ID=$(openstack floating ip create --description api.$CLUSTER_NAME.$BASE_DOMAIN $OPENSTACK_EXTERNAL_NETWORK --format value -c 'floating_ip_address' -c 'id')
+            LB_FIP=$(echo $LB_FIP_AND_ID |awk '{print $1}')
+            LB_FIP_UID=$(echo $LB_FIP_AND_ID |awk '{print $2}')
+            echo $LB_FIP_UID > /tmp/shared/LB_FIP_UID
+
+            INGRESS_FIP_AND_ID=$(openstack floating ip create --description ingress.$CLUSTER_NAME.$BASE_DOMAIN $OPENSTACK_EXTERNAL_NETWORK --format value -c 'floating_ip_address' -c 'id')
+            INGRESS_FIP=$(echo $INGRESS_FIP_AND_ID |awk '{print $1}')
+            INGRESS_FIP_UID=$(echo $INGRESS_FIP_AND_ID |awk '{print $2}')
+            echo $INGRESS_FIP_UID > /tmp/shared/INGRESS_FIP_UID
+
+
+            if [ -z "$DO_NOT_CREATE_DNS_RECORD" ]; then
+            echo "Creating DNS record for api.$CLUSTER_NAME.$BASE_DOMAIN. -> $LB_FIP (${LB_FIP_UID})"
+            cat > /tmp/artifacts/installer/api-record.json <<EOF
+            {
+            "Comment": "Create the public OpenShift API record",
+            "Changes": [{
+                    "Action": "UPSERT",
+                    "ResourceRecordSet": {
+                      "Name": "api.$CLUSTER_NAME.$BASE_DOMAIN.",
+                      "Type": "A",
+                      "TTL": 300,
+                      "ResourceRecords": [{"Value": "$LB_FIP"}]
+                    }
+            }]}
+            EOF
```

Comment:
Needs to reduce the indentation so that the heredoc marker is at the beginning of the line in the rendered script.

Reply:
yeap. fixing